examples/005_convenience/customize.py

```diff
@@ -6,82 +6,116 @@
-# plot.customize('model_line', 'label', None)  # Hide the model label in the info box.
-plot.customize('model_line', 'color', 'lightgreen')  # Set the model line color in the plot.
-plot.customize('model_line', 'linestyle', '-')  # Set the style of the model line in the plot.
-plot.customize('model_error_band', 'label', r'$\pm 1 \sigma$')  # Error band label in info box.
-# plot.customize('model_error_band', 'label', None)  # Hide error band label.
-plot.customize('model_error_band', 'color', 'lightgreen')  # Error band color in plot.
-# plot.customize('model_error_band', 'hide', True)  # Hide error band in plot and info box.
-
-# Available keywords can be retrieved with Plot.get_keywords(subplot_type) .
+# by the plot object.
+#
+# Available keywords can be retrieved with Plot.get_keywords(subplot_type).
 # subplot_type is for example 'data', 'model_line', or 'model_error_band'.
-
-# In addition to the customize method Plot has a few convenience methods for common operations:
-
-plot.x_range = (0.8, 6)  # Set the x range of the plot.
-plot.y_range = (1, 11)  # Set the y range of the plot.
-
-plot.x_label = 'My x axis 2'  # Overwrite the label of the x axis.
-plot.y_label = 'My y axis 2'  # Overwrite the label of the y axis.
-
-plot.x_scale = 'log'  # Set the x axis to a logarithmic scale.
-plot.y_scale = 'log'  # Set the y axis to a logarithmic scale.
-
-# Finally, perform the plot:
-plot.plot(ratio=True)
-plot.show()
+customized_plot.customize('data', 'marker', 'X')  # Set the data marker shape
+customized_plot.customize('data', 'markersize', 10)  # Set the data marker size
+customized_plot.customize('data', 'color', '#600E8F')  # Set the data marker color
+customized_plot.customize('data', 'ecolor', '#8F0BDB')  # Set the errorbar color
+customized_plot.customize('data', 'label', 'data label')  # Overwrite the data label in the info box
+
+customized_plot.customize('model_line', 'linestyle', ':')  # Set the linestyle for the model line
+customized_plot.customize('model_line', 'linewidth', 2)  # Set the line width for the model line
+customized_plot.customize('model_line', 'color', '#8F1B0E')  # Set the color of the model line
+customized_plot.customize('model_line', 'label',
+                          'model line label')  # Overwrite the model label in the info box
+
+customized_plot.customize('model_error_band', 'alpha',
+                          0.2)  # Set the alpha value (transparency) for the error band
+customized_plot.customize('model_error_band', 'linestyle',
+                          '--')  # Set the linestyle for the border of the error band
+customized_plot.customize('model_error_band', 'linewidth',
+                          3)  # Set the linewidth for the border of the error band
+customized_plot.customize('model_error_band', 'color', '#DB1F0B')  # Set the color of the error band
+customized_plot.customize('model_error_band', 'label',
+                          'model error band label')  # Set the label for the error band
```

Comment:
I would prefer to have comments above the code instead of having line breaks in the code.